operators/pkg/controller/common/certificates/http/reconcile.go

```diff
@@ -195,9 +194,10 @@ func ensureInternalSelfSignedCertificateSecretContents(
 	if shouldIssueNewHTTPCertificate(owner, namer, tls, secret, svcs, ca, rotationParam.RotateBefore) {
 		log.Info(
 			"Issuing new HTTP certificate",
-			"secret", secret.Name,
-			"namespace", owner.Namespace,
-			"name", owner.Name,
+			"namespace", secret.Namespace,
+			"secret_name", secret.Name,
+			"es_namespace", owner.Namespace,
+			"es_name", owner.Name,
```

Comment:
Recently Kibana is also using this package to issue HTTP certificate. So we should prefix the name and the namespace with `owner_` .